.github/workflows/test.yml

```diff
@@ -199,6 +203,8 @@ jobs:
           npm run load
 
       - name: Execute
+        env:
+          MONGODB_URI: mongodb://root:r00t@dtfs-submissions-data:27017/?authMechanism=DEFAULT&directConnection=true
```

Comment:
Per offline conversation -- otherwise feel free to resolve and lgtm

Reply:
Have deleted :) was from when I was testing something and didn't undo - good spot